installing/cluster-capabilities.adoc

```diff
@@ -0,0 +1,74 @@
+:_content-type: ASSEMBLY
+[id="cluster-capabilities"]
+= Cluster capabilities
+include::_attributes/common-attributes.adoc[]
+:context: cluster-capabilities
+
+toc::[]
+
+Cluster administrators can use cluster capabilities to enable or disable optional components prior to installation. Cluster administrators can enable cluster capabilities at anytime after installation.
+
+[NOTE]
+====
+Cluster administrators cannot disable a cluster capability after it is enabled.
+====
+
+include::modules/selecting-cluster-capabilities.adoc[leveloffset=+1]
+
+include::snippets/capabilities-table.adoc[]
+
+[role="_additional-resources"]
+.Additional resources
+* xref:../installing/installing_aws/installing-aws-customizations.adoc#installing-aws-customizations[Installing a cluster on AWS with customizations]
+* xref:../installing/installing_gcp/installing-gcp-customizations.adoc#installing-gcp-customizations[Installing a cluster on GCP with customizations]
+
+include::modules/explanation-of-capabilities.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+* xref:../operators/operator-reference.adoc#cluster-operator-reference[Cluster Operators reference]
+
+include::modules/cluster-bare-metal-operator.adoc[leveloffset=+2]
```

Comment:
ordering here seems to be by capability-definition-time, which doesn't seem like it will age well to me.  I'd expect alphabetical by section heading (as an obvious-ish way to say "order doesn't matter") or some attempt to group into use-case buckets (e.g. folks interested in storage probably care about these: ...).  But there's a table of contents in rendered docs, so the order is probably not all that important, and feel free to leave it as you have it if you like.

![image](https://user-images.githubusercontent.com/209920/211085905-3d9f6161-b0ad-4a21-a309-90abbecac5b7.png)

Reply:
Agreed - I've alphabetized them